azext_iot/central/models/preview/device.py

```diff
@@ -9,14 +9,14 @@
 
 class Device:
     def __init__(self, device: dict):
-        self.approved = device.get("approved")
+        self.approved = device.get("enabled") or device.get("approved")
         self.display_name = device.get("displayName")
         self.etag = device.get("etag")
         self.id = device.get("id")
-        self.instance_of = device.get("instanceOf")
+        self.instance_of = device.get("template") or device.get("instanceOf")
         self.provisioned = device.get("provisioned")
         self.simulated = device.get("simulated")
-        self.device_status = self._parse_device_status()
+        self._device_status = self._parse_device_status()
```

Comment:
is there a reason for "hiding" device status?

Reply:
device status is an internal concept not to be outputted to the user as per API specs. In previous versions the output dictionary was manually constructed to only specify valid keys. After code refactoring we now delegate to the framework (knack) the job of converting objects to dicts before output. The guideline in this case is to have "private" and "public" properties.

Reply:
Will there be any breaking changes/issues with respect to having this attribute now be private? This is assuming some customer decided to rely on device_status and now it is technically _device_status

Reply:
no breaking changes as previous version did not return device_status too. we did code refactoring to simplify things a little bit and this required non-public fields to be marked explicitly with underscore to prevent them appearing in the output